LinqToRest.Test/QueryProviderTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using Messerli.LinqToRest.Test.Stub;
+using Messerli.QueryProvider;
+using Messerli.ServerCommunication;
+using NSubstitute;
+using Xunit;
+
+namespace Messerli.LinqToRest.Test
+{
+    public class QueryProviderTest
+    {
+        [Fact]
+        public void ReturnsRestQuery()

```

Comment:
Why are all functions that return a mock object called `ResolveFoo`? Shouldn't they be called `MockFoo`?